src/providers/whatsapp/whatsapp.service.ts

```diff
@@ -99,4 +107,14 @@ export class WhatsappService {
 
     return response;
   }
+  async getContactList() {
+    const response = await this.client.getAllContacts();
+    return response;
+  }
+
+  async getContact(contact: Contacts) {
+    const { contactNumber } = contact;
+    const response = await this.client.getContact(contactNumber);
```

Comment:
Se eu bucar um numero assim: 96991934171 e depois 9691934171, ele retorna os dois contatos? ou retorna diferente? ou da erro em algum dos dois?

Reply:
`{
  "id": {
    "server": "c.us",
    "user": "5596991215506",
    "_serialized": "5596991215506@c.us"
  },
  "type": "in",
  "isBusiness": false,
  "isEnterprise": false,
  "statusMute": false,
  "labels": [],
  "formattedName": "+55 96 99121-5506",
  "isMe": false,
  "isMyContact": false,
  "isPSA": false,
  "isUser": true,
  "isWAContact": true,
  "profilePicThumbObj": {
    "eurl": null,
    "id": {
      "server": "c.us",
      "user": "5596991215506",
      "_serialized": "5596991215506@c.us"
    },
    "tag": null
  },
  "msgs": null
}`

Reply:
> com 2 noves

Reply:
`{
  "id": {
    "server": "c.us",
    "user": "559691215506",
    "_serialized": "559691215506@c.us"
  },
  "name": "Patrick",
  "shortName": "Patrick",
  "type": "in",
  "isBusiness": false,
  "isEnterprise": false,
  "statusMute": false,
  "labels": [],
  "formattedName": "Patrick",
  "isMe": false,
  "isMyContact": true,
  "isPSA": false,
  "isUser": true,
  "isWAContact": true,
  "profilePicThumbObj": {
    "eurl": "https://pps.whatsapp.net/v/t61.24694-24/187686320_929095707870900_4168986759395671043_n.jpg?ccb=11-4&oh=1c07ea4f2c51ed333f88d532a154877c&oe=60C11250",
    "id": {
      "server": "c.us",
      "user": "559691215506",
      "_serialized": "559691215506@c.us"
    },
    "img": "https://web.whatsapp.com/pp?e=https%3A%2F%2Fpps.whatsapp.net%2Fv%2Ft61.24694-24%2F187686320_929095707870900_4168986759395671043_n.jpg%3Fccb%3D11-4%26oh%3D1c07ea4f2c51ed333f88d532a154877c%26oe%3D60C11250&t=s&u=559691215506%40c.us&i=1622565042&n=nwlQoV79S3N1Iu1Ci3itai7UV3ePfgh1NosrEDezaGk%3D",
    "imgFull": "https://web.whatsapp.com/pp?e=https%3A%2F%2Fpps.whatsapp.net%2Fv%2Ft61.24694-24%2F187686320_929095707870900_4168986759395671043_n.jpg%3Fccb%3D11-4%26oh%3D1c07ea4f2c51ed333f88d532a154877c%26oe%3D60C11250&t=l&u=559691215506%40c.us&i=1622565042&n=nwlQoV79S3N1Iu1Ci3itai7UV3ePfgh1NosrEDezaGk%3D",
    "raw": null,
    "tag": "1622565042"
  },
  "msgs": null
}`

Reply:
com 1 nove

Reply:
quando tu pesquisa com 1 nove ele gera mais informações e a chave isMyContact é true, quando tu pesquisa com 2 noves, gera menos informações e a chave retorna false

Reply:
tu usou o mesmo numero, o segundo que tu buscou ali é diferente 559691215506

Reply:
entendi

Reply:
Sera que nao tem nenhum parametro pra passar pro venom tratar isso?

Reply:
não encontrei